src/execution/utils.ts

```diff
@@ -454,9 +487,49 @@ async function submitUserOp(
   sourceChain: Chain,
   targetChain: Chain,
   userOp: UserOperation,
-  orderPath: OrderPath,
   signature: Hex,
+  orderPath?: OrderPath,
 ) {
+  if (!orderPath) {
+    const publicClient = createPublicClient({
+      chain: sourceChain,
+      transport: http(),
+    })
+    const bundlerClient = getBundlerClient(config, publicClient)
+    const hash = await bundlerClient.request({
+      method: 'eth_sendUserOperation',
+      params: [
+        {
+          sender: userOp.sender,
+          nonce: toHex(userOp.nonce),
+          factory: userOp.factory,
+          factoryData: userOp.factoryData,
+          callData: userOp.callData,
+          callGasLimit: toHex(userOp.callGasLimit),
+          verificationGasLimit: toHex(userOp.verificationGasLimit),
+          preVerificationGas: toHex(userOp.preVerificationGas),
+          maxPriorityFeePerGas: toHex(userOp.maxPriorityFeePerGas),
+          maxFeePerGas: toHex(userOp.maxFeePerGas),
+          paymaster: userOp.paymaster,
+          paymasterVerificationGasLimit: userOp.paymasterVerificationGasLimit
+            ? toHex(userOp.paymasterVerificationGasLimit)
+            : undefined,
+          paymasterPostOpGasLimit: userOp.paymasterPostOpGasLimit
+            ? toHex(userOp.paymasterPostOpGasLimit)
+            : undefined,
+          paymasterData: userOp.paymasterData,
+          signature,
+        },
+        entryPoint07Address,
+      ],
+    })
```

Comment:
`viem` doesn't let us to use [`sendUserOperation`](https://viem.sh/account-abstraction/actions/bundler/sendUserOperation) w/o specifying `account`/`calls`, so have to prepare the request manually